src/pages/login/index.js

```diff
@@ -6,48 +6,78 @@ import CredentialsCard from "../../components/credentials";
 import "./login.css";
 
 const Login = () => {
-	const { onLogin } = useAuth();
-	const [formData, setFormData] = useState({ email: "", password: "" });
-
-	const onChange = (e) => {
-		const { name, value } = e.target;
-		setFormData({ ...formData, [name]: value });
-	};
-
-	return (
-		<div className="bg-blue login credentialpage">
-			<CredentialsCard
-				title="Login"
-				socialLinksTitle="Or log in with"
-				altButtonTitle="Need an account?"
-				altButtonLink="/register"
-				altButtonText="Sign up"
-			>
-				<div className="login-form">
-					<form>
-						<TextInput
-							value={formData.email}
-							onChange={onChange}
-							name="email"
-							label={"Email *"}
-						/>
-						<TextInput
-							value={formData.password}
-							onChange={onChange}
-							name="password"
-							label={"Password *"}
-							type={"password"}
-						/>
-					</form>
-					<Button
-						text="Log in"
-						onClick={() => onLogin(formData.email, formData.password)}
-						classes="green width-full"
-					/>
-				</div>
-			</CredentialsCard>
-		</div>
-	);
+  const { onLogin } = useAuth();
+  const [formData, setFormData] = useState({ email: "", password: "" });
+  const [errors, setErrors] = useState({ email: "", password: "" });
+
+  const isValidEmail = (emailAddress) => {
+    return emailAddress.trim() !== "";
```

Comment:
not sure this is needed, you can just call trim regardless and avoid the user having to figure out that they have empty spaces at the start or end of the string

you could at least check that an @ and a . character are present

Reply:
Noted, will update it